.mypy.ini

```diff
@@ -0,0 +1,61 @@
+[mypy]
+python_version = 3.8
+warn_return_any = True
+warn_unused_configs = True
+
+[mypy-addfips.*]
+ignore_missing_imports = True
```

Comment:
My understanding is that these are all packages that don't do typing themselves, and that have no stub packages available. I mostly copied it from the `.mypy.ini` file in the PUDL repo, which I think was created originally by @ezwelty.

Reply:
Actually it looks like some of them have implemented typing since this list was created. I see `pytest` has been typed since 6.0. I guess we should figure out which of these are actually required and which are obsolete.